### DISCUSSION TITLE
Installed iiab on laptop but dont know what to do next

### DISCUSSION THREAD

**Discussion by Netscapevos** [iiab/iiab#3740] [Q&A]
after installation it created a new user iiab - admin now i can log in but what do i do to use iiab

**Comment by tim-moody** [comment]
the install probably created a hotspot (if the laptop wifi supports it). a device with wifi can connect to that hotspot named Internet in a Box and get to the home page at http://box.lan. Next you will need to add content using  http://box.lan/admin.
